app/src/main/res/layout-land/fragment_top_up.xml

```diff
@@ -282,6 +282,36 @@
           app:layout_constraintTop_toBottomOf="@id/container"
           />
 
+      <TextView
+          android:id="@+id/offer_wall_msg"
+          style="@style/TopUpTextStyle.Small.BonusText"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="@dimen/big_margin"
+          android:layout_marginTop="11dp"
```

Comment:
Google recommends using multiples of 8 in margins, or padding, or size.

I recommend changing the margin